lib/ui/page/home/page/chat/view.dart

```diff
@@ -1413,65 +1413,85 @@ class ChatView extends StatelessWidget {
-                                c.selected.every((e) {
-                                  if (e is ChatMessageElement) {
-                                    return e.item.value.author.id == c.me &&
-                                        c.chat?.chat.value.isRead(
-                                              e.item.value,
-                                              c.me,
-                                            ) ==
-                                            false;
-                                  } else if (e is ChatForwardElement) {
-                                    return e.authorId == c.me &&
-                                        c.chat?.chat.value.isRead(
-                                              e.forwards.first.value,
-                                              c.me,
-                                            ) ==
-                                            false;
-                                  } else if (e is ChatInfoElement) {
-                                    return false;
-                                  } else if (e is ChatCallElement) {
-                                    return false;
-                                  }
+                            final bool isMonolog =
+                                c.chat?.chat.value.isMonolog ?? false;
+                            final bool deletable = c.selected.every((e) {
+                              if (e is ChatMessageElement) {
+                                return e.item.value.author.id == c.me &&
+                                    c.chat?.chat.value.isRead(
+                                          e.item.value,
+                                          c.me,
+                                        ) ==
+                                        false;
+                              } else if (e is ChatForwardElement) {
+                                return e.authorId == c.me &&
+                                    c.chat?.chat.value.isRead(
+                                          e.forwards.first.value,
+                                          c.me,
+                                        ) ==
+                                        false;
+                              } else if (e is ChatInfoElement) {
+                                return false;
+                              } else if (e is ChatCallElement) {
+                                return false;
+                              }
 
-                                  return false;
-                                });
+                              return false;
+                            });
+
+                            bool deleteForAll = false;
+
+                            final result =
+                                await MessagePopup.alert(
+                                  c.selected.length > 1
+                                      ? 'label_delete_messages'.l10n
+                                      : 'label_delete_message'.l10n,
```

Comment:
I see you've decided to keep the old l10n phrases instead of using the new ones provided by the UI/UX team. Why is that? Such things should always be discussed with the team, especially since every single decision was made on purpose by the whole UI/UX team during intense discussions.

<img width="798" alt="Screenshot 2025-06-24 at 14 54 59" src="https://github.com/user-attachments/assets/6364e1c8-4afb-423d-acb2-dce5e2a7d1ba" />